files/20-wwan0.network

```diff
@@ -3,6 +3,8 @@ Name=wwan0
 
 [Network]
 DHCP=ipv4
+DNS=8.8.8.8
+DNS=8.8.4.4
```

Comment:
FYI, you probably already know but just in case you aren't, you'll need to remove this for any Pis on the NSSTC WiFI because AFAIK it requires using its own DNS servers

Reply:
Since this is the wwan network config, this shouldn't apply to the NSSTC wifi...right?

Reply:
Oh, of course—silly me. :facepalm: Before it had the DNS set up in a global config file, but with `systemd-networkd` everything is per-interface. Thanks.